calling-apis/managing-asynchronous-calls.md

```diff
@@ -0,0 +1,277 @@
+# Managing Asynchronous Calls
+
+## Goal
+
+The goal for this lesson is to witness how asynchronous code can effect our data management.
+
+We should walk away from this lesson with one strategy to manage asynchronous code. We should also be able to question and research what other strategies may exist.
+
+## Experiment: Making Multiple Calls
+
+Raffy is building a map web app. He wants to run an experiment!
+
+He wants to make two API requests to the [LocationIQ APIs](https://locationiq.com/docs):
+
+1. First, he wants to make a request to the Forward Geocoding API to find the latitude and longitude of "Seattle, Washington, USA."
+1. Then, he wants to make a request to the Reverse Geocoding API with the latitude and longitude he found, and see if the results come back as "Seattle, Washington, USA."
+
+He decides to run his experiment with [this code](https://replit.com/@adacore/Managing-Asynchronous-Requests)!
+
+<!-- prettier-ignore-start -->
+```js
+const axios = require("axios");
+
+const YOUR_API_KEY = 'YOUR API KEY';
```

Comment:
fixes quotes and gets the key from a secret

these changes and the following should get applied to the replit

```suggestion
const axios = require('axios');

const LOCATIONIQ_KEY = process.env['api_key'];
```